packages/react/README.md

```diff
@@ -303,6 +303,22 @@ module.exports = {
 };
 ```
 
+#### Clean external directories
```

Comment:
I think we should only add a section to the `@neutrinojs/clean` docs rather than the top-level presets too.

Reply:
Thanks for the input. I have updated it.